deploy/template.yaml

```diff
@@ -25,9 +25,6 @@ Conditions:
       - !Ref ProvisionedConcurrentExecutions
       -  0
 
-  SendLogsToSplunk: !Not
-    - !Equals [ !Ref Environment, integration ]
-
```

Comment:
Was the intent of this not so it could be used in the future as a switch?

Reply:
nah it was just a quick bypass